cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -903,422 +903,983 @@ frappe.pages['payroll-configuratio'].on_page_load = function(wrapper) {
+								// If multi_insert is 1, no need to fetch, create the dialog immediately
+								createAndShowDialog();
+							}
+
+
+							// console.log(Child_custom_field,"Child_custom_fieldChild_custom_fieldChild_custom_fieldChild_custom_field")
+							
+							function createAndShowDialog() {
+								console.log(Child_custom_field, "Child Custom Fields");
+								console.log(custom_field, "Custom Fields");
+							
+								let dialogFields = [
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'dialogFields' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)